packages/eslint-plugin-putout/lib/keyword-spacing/index.js

```diff
@@ -3,7 +3,7 @@
 const {
     isTryStatement,
     isSwitchStatement,
-} = require('putout').types;
+} = require('../putout-wrapper')().types;
```

Comment:
Could you please use [tryrequire](https://www.npmjs.com/package/try-catch) similar to [packages/putout/putout/lib/cli/eslint.js](https://github.com/coderaiser/putout/blob/v11.0.0/packages/putout/lib/cli/eslint.js#L7).